### ISSUE TITLE
[Suggestion] randomized options

### ISSUE THREAD

**Issue by ErtasVideos** [LagoLunatic/wwrando#50]
i wonder if its possible to have all randomized option

**Comment by LagoLunatic** [comment]
You mean like a button that randomizes which options are checked off?

**Comment by ErtasVideos** [comment]
yeah like that example: you randomize options and you see which options are on and off
